tests/testthat/test-derive_summary_records.R

```diff
@@ -193,3 +193,26 @@ test_that("derive_summary_records Test 6: test missing values with dataset_ref",
     keys = c("subj", "visit", "seq", "type")
   )
 })
+
+## Test 6: error if no summary function ----
+test_that("derive_summary_records Test 6: error if no summary function", {
```

Comment:
Hey yo--sorry I might be getting a bit dense here.

Does this function only take summary functions?  If so, I was thinking on this more when I was writing unit tests in my PR, i.e. should we say something more explicit to the users that a summary/aggregate-like function is expected for this function.  I know we use aggregate functions in the title but wondering if docs need to be more explicit???

Also - is the error appropriate?  The test says "error if no summary function" but the error in the snapshot is for AVISIT??

Apologies if I am failing to connect the dots.   

<img width="881" height="348" alt="image" src="https://github.com/user-attachments/assets/4a281726-6acc-46b9-80aa-89892f13afd6" />